src/main/java/frc/robot/controls/Driver.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.controls;
+
+import frc.robot.commands.elevator.DoNothing;
+import frc.robot.constants.OIConstants;
+import frc.robot.subsystems.Drivetrain;
+import lib.controllers.GameController;
+import lib.controllers.GameController.Button;
+
+public class Driver {
+  private static GameController driver = new GameController(OIConstants.kDriverJoy);
+
+  /**
+   * Configures all the driver controls, which are the default controls for the robot.
+   */
+  public static void configureControls(Drivetrain drive) {
+    
+    // example button binding implementation
+    driver.get(Button.A).onTrue(new DoNothing());
+  }
+}
```

Comment:
This should not exist we removed normal Driver in a previous PR to give multiple controller options to the driver